### DISCUSSION TITLE
What's the best way a coding beginner can help contribute to GitHub?

### DISCUSSION THREAD

**Discussion by oo7Robin** [community/community#21686] [New to GitHub]
<p>Where do I go for help? What’s the best way for an inexperienced programmer/student to help contribute to GitHub (from an employer’s perspective)? </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/oo7robin">@oo7robin</a>, </p>
<p>Welcome and thanks for being here! I’d recommended finding an Open Source Software project that’s related to your field of studies or the goals you might have for employment. </p>
<p>Here is a list of resources that might help:</p>
<ul>
<li><a href="https://www.wired.com/story/nsa-ghidra-open-source-tool/" rel="nofollow noopener">Ghidra</a></li>
<li><a href="https://techbeacon.com/app-dev-testing/57-open-source-app-sec-tools-guide-free-application-security-software" title="57 open source app sec tools: A guide to free application security software" rel="nofollow noopener">https://techbeacon.com/app-dev-testing/57-open-source-app-sec-tools-guide-free-application-security-software</a></li>
<li><a href="https://github.community/t/best-practices-for-open-source-software-contributions/10209">Best practices for OSS contributions</a></li>
</ul>

**Comment by oo7Robin** [comment]
<p>Thanks Andrea, I really appreciate your help :slight_smile:</p>
<p>Kind Regards,</p>
<p>Robin.</p>

**Comment by ghost** [comment]
<p>HI <a href="/t5/user/viewprofilepage/user-id/40651">@oo7Robin</a>, </p>
<p>Have you found something to work on?</p>
<p>Hi Andrea, </p>
<p>I am looking for projects on python, is there a good place for Python?</p>
<p>Thx</p>
